bin/console

```diff
@@ -4,4 +4,9 @@ require 'bundler/setup'
 require 'wannabe_bool'
 
 require 'pry'
```

Comment:
I'd suggest to move this line to inside the condition if Pry is defined.

Reply:
Yep!  Thank you for that comment.  After using this PR I noticed some issues.  Will followup with a fix.